src/precice/impl/MeshContext.hpp

```diff
@@ -35,7 +35,12 @@ struct MeshContext {
   bool allowDirectAccess = false;
 
   /// setMeshAccessRegion may only be called once per mesh(context)
-  bool accessRegionDefined = false;
+  /// putting this into the mesh context means that we can only call
+  /// this once, regardless of combinations with just-in-time mappings
+  /// or multiple such mappings
+  /// if multiples are desired, we would need to shift this into the
+  /// MappingContext
```

Comment:
Odd line breaking

Reply:
these are individual paragraphs. I can put another empty line in between if this makes it less odd?

Reply:
Why not a multiline comment? You could also add bullet points in the multiline comment to have a better formatting? Right now I really do not understand the rationale behind the line breaks.

Reply:
I removed the line break